sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/GenerateOrdering.scala

```diff
@@ -118,7 +118,45 @@ object GenerateOrdering extends CodeGenerator[Seq[SortOrder], Ordering[InternalR
           }
       """
     }.mkString("\n")
-    comparisons
+
+    /*
```

Comment:
@davies could you take a quick look here:

this PR tries to break a huge generated method into smaller pieces. @kiszk @ueshin and I were discussing whether we should:
- (a) just do the breaking-ups case by case, like in this PR
- or (b) expand `CodeGenerator.splitExpressions(...)` to generally support this breaking-ups, like in the POC https://github.com/lw-lin/spark/commit/d0c11984e9aa84f68b9ebdd21317f629d759ae86

@davies could you advise on this? thanks!